DxDispatch/src/dxdispatch/ModuleInfo.cpp

```diff
@@ -0,0 +1,116 @@
+#include "pch.h"
+#include "ModuleInfo.h"
+#include "config.h"
+
+#ifdef _WIN32
+
+#ifdef _GAMING_XBOX
+// QueryUnbiasedInterruptTime is not declared in WINAPI_PARTITION_GAMES
+// Undefining _APISETREALTIME_ will avoid a few win32_helpers declarations
+// that won't work in WINAPI_PARTITION_GAMES.
+#undef _APISETREALTIME_
+#endif
+
+#include <wil/win32_helpers.h>
+
+struct LanguageAndCodePage
+{
+    WORD language;
+    WORD codePage;
+};
+
+std::optional<ModuleInfo> GetModuleInfo(std::string moduleName)
```

Comment:
>std::string [](http://example.com/codeflow?start=40&length=11)

[minor] Propose `gsl::czstring`, since this is only ever passed string literals anyway - no need for alloc.